src/yarp.c

```diff
@@ -4548,7 +4550,7 @@ static yp_token_type_t
 lex_numeric_prefix(yp_parser_t *parser) {
     yp_token_type_t type = YP_TOKEN_INTEGER;
 
-    if (peek_at(parser, -1) == '0') {
+    if (peek_ahead(parser, -1) == '0') {
```

Comment:
Is it weird to "peek ahead" backwards?

Reply:
Great point! I just updated it to `peek_offset`